bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/util/Flags.java

```diff
@@ -34,6 +34,7 @@
     public static final int OPTIONAL = 8192;
     public static final int RESOURCE = 262144;
     public static final int SERVICE = 524288;
+    public static final int REMOTE = 65536;
```

Comment:
Can we move this to L35 so that the order is preserved?

Reply:
Fixed